marimo/_server/export/utils.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import annotations
 
 import asyncio
+from multiprocessing import Process
```

Comment:
just confirming - does this import make wasm fail? i think we import this file into wasm

Reply:
No it doesn't, the multiprocessing module is available in Pyodide, even though some of its features don't work.

https://marimo.app/l/hzanca